### PR TITLE
Changed the UI of project card

### PR DESCRIPTION
**Pull request by AKC068**
As per provided reference link, I did all the changes. Please review once.
I've also provided a demo video that how it is working in last comment.

### CODE REVIEW

**Comment by AKC068** [comment]
What should I change in this according to you?

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Add appropriate padding in this so that cards won't stick together.

**Comment by AKC068** [comment]
@anushbhatia sure...

**Comment by AKC068** [comment]
I've done all the changes as per requested and required.
- Added padding to cards.
- Added extra animation on hover.
- Added AOS animation also.
- And these cards are responsive in every view and working properly.

@anushbhatia please once review all the changes and let me know if it needs more enhancement or anything different.
@smaranjitghose

File: assets/css/project.css [hunk 1]
```diff
@@ -35,7 +35,17 @@ html[data-theme = 'dark']
 body{
```
Comment (smaranjitghose) [inline assets/css/project.css:35]: what is the `.swp` file for?

Comment (AKC068) [reply assets/css/project.css:35]: I'm checking.
I don't know how it was added from my side

Comment (AKC068) [reply assets/css/project.css:35]: @smaranjitghose its done.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Try creating a new PR cause this might be having bugs with it.

**Comment by AKC068** [comment]
ok

**Comment by smaranjitghose** [comment]
@AKC068 as requested by @anushbhatia to continue in another PR, I am closing this one

**Comment by AKC068** [comment]
@smaranjitghose can you please assign me issue #743 as per requested.

**Pull request closed without merging**